spec/models/person_spec.rb

```diff
@@ -27,24 +22,25 @@
     end
 
     it 'has too short name' do
-      new_person.name = 'Per'
+      new_person.name = 'P'
       expect(new_person).not_to be_valid
     end
   end
 
   describe 'CRUD operations' do
     let!(:user) do
-      User.create(id: 1, name: 'User', email: 'user@mail.ru', password: '123456', password_confirmation: '123456')
+      User.create(id: 2, name: 'User', email: 'user@mail.ru', password: '123456', password_confirmation: '123456')
```

Comment:
Created unique id, because I've created anothoer User with this id in User model rspec